packages/babel-preset-env/test/fixtures/bugfixes-always-enabled/chrome-96-static-class-fields/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "targets": { "chrome": 96 },
```

Comment:
Do you know which version of Chrome it was fixed in? The only Chrome 97 download I can find has the bug.

Reply:
compat-table should cover every version. if there's no node or chrome or edge releases that contain the bug, then there's no point in compat-table or babel fixing it; if there's any, then we must.

is this in any node versions?

Reply:
> Do you know which version of Chrome it was fixed in

The fix has been landed in V8 Version 9.7.106.21. however V8 9.7.106.21 was not bundled to Chrome 97.0.4692.99 ([changelog](https://chromium.googlesource.com/chromium/src/+log/97.0.4692.71..97.0.4692.99?pretty=fuller&n=10000)). Since then the stable channel was updated to 98 so probably they don't bother to release a fix for Chrome 97. So we can apply the bugfix for Chrome 97.

> is this in any node versions?

AFAIK no, the upstream fix https://github.com/nodejs/node/commit/e23e345b6c6b66c7fac5c76741bbfef722953e3a is cherry-picked to the bundled V8 versions in Node.js. So we should not apply the bugfix plugin for node targets.

Reply:
Ok, so it’s just for chrome 97 then. To answer the original question, we should be explicit about which chrome version it’s fixed in, but we should also only be displaying the latest version.

Reply:
PR updated with https://github.com/kangax/compat-table/pull/1796. Note that we still load the bugfix plugin when targeting chrome 96, because it means "my code should work in every Chrome version >= 96" and thus it also included chrome 97.